jcl/src/java.base/share/classes/java/lang/invoke/ArrayVarHandle.java

```diff
@@ -163,66 +150,69 @@ private static final IllegalArgumentException newIllegalArgumentException(Class<
 		}
 
 		static final class OpObject extends ArrayVarHandleOperations {
+			private static final int OFFSET = _unsafe.arrayBaseOffset(Object[].class);
```

Comment:
A better name for this class of fields would be 'BASE_OFFSET'.

Reply:
Fixed